src/main/cpp/src/query_operations/variant_operations.cc

```diff
@@ -582,6 +583,40 @@ void GA4GHOperator::operate(Variant& variant, const VariantQueryConfig& query_co
         copy_field(remapped_field, orig_field);
         if (remapped_field.get() && remapped_field->is_valid()) {   //Not null
           auto curr_ploidy = m_ploidy[curr_call_idx_in_variant];
+	  if (length_descriptor.is_length_genotype_dependent()
```

Comment:
Looks good. Is there any way to refactor the error processing out of this method just for readability? The GA4GHOperator::operate contains more than 100 lines of code.

Reply:
As of now, that part of the code isn't used anywhere else. If it is in future, I'll pull it out